--- vendor/ircmaxell/php-llvm/lib/Value/Instruction.php
+++ vendor/ircmaxell/php-llvm/lib/Value/Instruction.php
@@ -47,4 +47,6 @@ interface Instruction extends Value {
 
     public function addDestination(BasicBlock $dest): void;
 
+    public function addIncoming(Value $value, BasicBlock $block): void;
+
 }
--- vendor/ircmaxell/php-llvm/lib/LLVMAbstract/Value/Instruction.php
+++ vendor/ircmaxell/php-llvm/lib/LLVMAbstract/Value/Instruction.php
@@ -85,4 +85,13 @@ class Instruction extends Value implements CoreValue\Instruction {
         $this->llvm->lib->LLVMAddDestination($this->value, $dest->block);
     }
 
+    public function addIncoming(CoreValue $value, CoreBasicBlock $block): void {
+        $this->llvm->lib->LLVMAddIncoming(
+            $this->value,
+            $this->llvm->lib->makeArray(\llvm\LLVMValueRef_ptr::class, [$value->value]),
+            $this->llvm->lib->makeArray(\llvm\LLVMBasicBlockRef_ptr::class, [$block->block]),
+            1
+        );
+    }
+
 }
