--- vendor/ircmaxell/php-types/lib/PHPTypes/InternalArgInfo.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/InternalArgInfo.php
@@ -1467,7 +1467,7 @@
         'expect_popen' => ['resource', 'command' => 'string'],
         'exp' => ['float', 'number' => 'float'],
-        'explode' => ['string[]', 'separator' => 'string', 'str' => 'string', 'limit=' => 'int'],
+        'explode' => ['array', 'separator' => 'string', 'str' => 'string', 'limit=' => 'int'],
         'expm1' => ['float', 'number' => 'float'],
         'extension_loaded' => ['bool', 'extension_name' => 'string'],
         'extract' => ['int', 'var_array' => 'array', 'extract_type=' => 'int', 'prefix=' => 'string'],
