--- vendor/ircmaxell/php-types/lib/PHPTypes/InternalArgInfo.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/InternalArgInfo.php
@@ -3723,7 +3723,7 @@
         'ldap_get_attributes' => ['array', 'link' => '', 'result_entry' => ''],
         'ldap_get_dn' => ['string', 'link' => '', 'result_entry' => ''],
         'ldap_get_entries' => ['array', 'link' => '', 'result' => ''],
-        'ldap_get_option' => ['bool', 'link' => '', 'option' => 'int', 'retval' => ''],
+        'ldap_get_option' => ['bool', 'link' => '', 'option' => 'int', '&retval' => ''],
         'ldap_get_values' => ['array', 'link_identifier' => '', 'result_entry_identifier' => '', 'attribute' => 'string'],
         'ldap_get_values_len' => ['array', 'link' => '', 'result_entry' => '', 'attribute' => 'string'],
         'ldap_list' => ['resource', 'link' => 'resource|array', 'base_dn' => 'string', 'filter' => 'string', 'attrs=' => 'array', 'attrsonly=' => 'int', 'sizelimit=' => 'int', 'timelimit=' => 'int', 'deref=' => 'int'],
