--- vendor/ircmaxell/php-types/lib/PHPTypes/InternalArgInfo.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/InternalArgInfo.php
@@ -7150,7 +7150,7 @@
         'SplFixedArray::__construct' => ['SplFixedArray', 'size' => 'int'],
         'SplFixedArray::count' => ['int'],
         'SplFixedArray::current' => [''],
-        'SplFixedArray::fromArray' => ['object', 'data' => 'array', 'save_indexes=' => 'bool'],
+        'SplFixedArray::fromArray' => ['SplFixedArray', 'array' => 'array', 'preserveKeys=' => 'bool'],
         'SplFixedArray::getSize' => ['int'],
         'SplFixedArray::key' => ['int'],
         'SplFixedArray::next' => [''],
