--- vendor/ircmaxell/php-types/lib/PHPTypes/TypeReconstructor.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/TypeReconstructor.php
@@ -246,6 +246,7 @@
 
                 return false;
             case 'Expr_Exit':
             case 'Iterator_Reset':
                 return [Type::null()];
+            case 'Expr_Throw':
+                return [Type::never()];
             case 'Iterator_Valid':
