--- vendor/ircmaxell/php-types/lib/PHPTypes/InternalArgInfo.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/InternalArgInfo.php
@@ -1852,7 +1852,7 @@
         'get_magic_quotes_gpc' => ['int'],
         'get_magic_quotes_runtime' => ['int'],
-        'get_meta_tags' => ['array', 'filename' => 'string', 'use_include_path=' => 'bool'],
+        'get_meta_tags' => ['array|false', 'filename' => 'string', 'use_include_path=' => 'bool'],
         'getmxrr' => ['bool', 'hostname' => 'string', 'mxhosts' => 'array', 'weight=' => 'array'],
         'getmygid' => ['int'],
