--- vendor/ircmaxell/php-llvm/lib/LLVMAbstract/MemoryBuffer.php
+++ vendor/ircmaxell/php-llvm/lib/LLVMAbstract/MemoryBuffer.php
@@ -5,8 +5,10 @@ namespace PHPLLVM\LLVMAbstract;
 use PHPLLVM\Context as CoreContext;
 use PHPLLVM\MemoryBuffer as CoreMemoryBuffer;
 
+use FFI;
 use llvm\llvm as lib;
 use llvm\LLVMMemoryBufferRef;
 use llvm\LLVMModuleRef;
+use llvm\string_ptr;
 
 class MemoryBuffer implements CoreMemoryBuffer {
 
@@ -32,24 +34,24 @@ class MemoryBuffer implements CoreMemoryBuffer {
     }
 
     public function parseBitcode(CoreContext $context): Module {
         $error = new string_ptr(FFI::addr(FFI::new('char*')));
-        $module = new LLVMModuleRef($this->lib->getFFI()->new('LLVMModuleRef'));
+        $module = new LLVMModuleRef($this->llvm->lib->getFFI()->new('LLVMModuleRef'));
         if (!$this->llvm->fromBool($this->llvm->lib->LLVMParseBitcodeInContext($context->context, $this->buffer, $module->addr(), $error))) {
-            $message = $error->deref(0);
-            $this->disposeMessage($error->deref(0));
+            $message = $error->deref(0)->toString();
+            $this->llvm->disposeMessage($error->deref(0));
             throw new \RuntimeException("Bitcode parsing failed due to $message");
         }
-        return $this->llvm->factory->module($context, $mdoule);
+        return $this->llvm->factory->module($context, $module, 'preg_match_runtime');
     }
 
     public function getBitcodeModule(CoreContext $context): Module  {
         $error = new string_ptr(FFI::addr(FFI::new('char*')));
-        $module = new LLVMModuleRef($this->lib->getFFI()->new('LLVMModuleRef'));
+        $module = new LLVMModuleRef($this->llvm->lib->getFFI()->new('LLVMModuleRef'));
         if (!$this->llvm->fromBool($this->llvm->lib->LLVMGetBitcodeModuleInContext($context->context, $this->buffer, $module->addr(), $error))) {
-            $message = $error->deref(0);
-            $this->disposeMessage($error->deref(0));
+            $message = $error->deref(0)->toString();
+            $this->llvm->disposeMessage($error->deref(0));
             throw new \RuntimeException("Bitcode module fetch failed due to $message");
         }
-        return $this->llvm->factory->module($context, $mdoule);
+        return $this->llvm->factory->module($context, $module, 'preg_match_runtime');
     }
 }
