--- vendor/ircmaxell/php-types/lib/PHPTypes/Type.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/Type.php
@@ -418,3 +418,3 @@
         if (! preg_match($regex, $decl)) {
-            throw new \RuntimeException("Unknown type declaration found: ${decl}");
+            throw new \RuntimeException("Unknown type declaration found: {$decl}");
         }
@@ -550,3 +550,3 @@
             if (! in_array(substr($decl, 0, 1), ['|', '&'], true)) {
-                throw new \RuntimeException("Unknown position of combinator: ${decl}");
+                throw new \RuntimeException("Unknown position of combinator: {$decl}");
             }
@@ -566,3 +566,3 @@
             } else {
-                throw new \RuntimeException("No combinator found: ${decl}");
+                throw new \RuntimeException("No combinator found: {$decl}");
             }
@@ -569,3 +569,3 @@
             if ($pos === 0) {
-                throw new \RuntimeException("Unknown position of combinator: ${decl}");
+                throw new \RuntimeException("Unknown position of combinator: {$decl}");
             }
@@ -582,2 +582,2 @@
-        throw new \RuntimeException("Unknown combinator ${combinator}");
+        throw new \RuntimeException("Unknown combinator {$combinator}");
     }
--- vendor/ircmaxell/php-types/lib/PHPTypes/State.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/State.php
@@ -224,2 +224,2 @@
-                echo "Could not find parent ${extends}\n";
+                echo "Could not find parent {$extends}\n";
             }
