--- vendor/ircmaxell/php-types/lib/PHPTypes/InternalArgInfo.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/InternalArgInfo.php
@@ -5320,10 +5320,10 @@
         'openssl_csr_get_subject' => ['', 'csr' => '', 'use_shortnames=' => 'bool'],
         'openssl_csr_new' => ['bool', 'dn' => 'array', '&privkey' => '', 'configargs=' => 'array', 'extraattribs=' => 'array'],
         'openssl_csr_sign' => ['resource', 'csr' => '', 'x509' => '', 'priv_key' => '', 'days' => 'int', 'config_args=' => 'array', 'serial=' => 'int'],
-        'openssl_decrypt' => ['string', 'data' => 'string', 'method' => 'string', 'password' => 'string', 'options=' => 'int', 'iv=' => 'string'],
+        'openssl_decrypt' => ['string|false', 'data' => 'string', 'cipher_algo' => 'string', 'passphrase' => 'string', 'options=' => 'int', 'iv=' => 'string', 'tag=' => '?string', 'aad=' => '?string'],
         'openssl_dh_compute_key' => ['string', 'pub_key' => 'string', 'dh_key' => ''],
         'openssl_digest' => ['string', 'data' => 'string', 'method' => 'string', 'raw_output=' => 'bool'],
-        'openssl_encrypt' => ['string', 'data' => 'string', 'method' => 'string', 'password' => 'string', 'options=' => 'int', 'iv=' => 'string'],
+        'openssl_encrypt' => ['string|false', 'data' => 'string', 'cipher_algo' => 'string', 'passphrase' => 'string', 'options=' => 'int', 'iv=' => 'string', '&tag=' => 'string', 'aad=' => '?string', 'tag_length=' => 'int'],
         'openssl_error_string' => [''],
         'openssl_free_key' => ['', 'key_identifier' => ''],
         'openssl_get_cert_locations' => ['array'],
