--- vendor/ircmaxell/php-types/lib/PHPTypes/InternalArgInfo.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/InternalArgInfo.php
@@ -108,7 +108,7 @@
         'array' => ['array', '...values=' => ''],
         'array_change_key_case' => ['array', 'input' => 'array', 'case=' => 'int'],
         'array_chunk' => ['array', 'input' => 'array', 'size' => 'int', 'preserve_keys=' => 'bool'],
-        'array_combine' => ['array', 'keys' => 'array', 'values' => 'array'],
+        'array_combine' => ['array|false', 'keys' => 'array', 'values' => 'array'],
         'array_count_values' => ['array', 'input' => 'array'],
         'array_diff' => ['array', 'arr1' => 'array', 'arr2' => 'array', '...=' => 'array'],
         'array_diff_assoc' => ['array', 'arr1' => 'array', 'arr2' => 'array', '...=' => 'array'],
