--- vendor/ircmaxell/php-types/lib/PHPTypes/InternalArgInfo.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/InternalArgInfo.php
@@ -3765,7 +3765,7 @@
         'LimitIterator::seek' => ['', 'position' => 'int'],
         'LimitIterator::valid' => ['bool'],
         'linkinfo' => ['int', 'filename' => 'string'],
-        'link' => ['int', 'target' => 'string', 'link' => 'string'],
+        'link' => ['bool', 'target' => 'string', 'link' => 'string'],
         'list' => ['array', '...varname' => ''],
         'litespeed_request_headers' => ['array'],
         'litespeed_response_headers' => ['array'],
