--- vendor/ircmaxell/php-types/lib/PHPTypes/TypeReconstructor.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/TypeReconstructor.php
@@ -728,7 +728,7 @@
         if ($type instanceof Op\Type\Mixed_) {
             return Type::mixed();
         }
         if ($type instanceof Op\Type\Nullable) {
-            (new Type(Type::TYPE_UNION, [
+            return (new Type(Type::TYPE_UNION, [
                 $this->resolveOpType($type->subtype),
                 new Type(Type::TYPE_NULL),
             ]))->simplify();
