--- vendor/ircmaxell/php-types/lib/PHPTypes/Type.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/Type.php
@@ -305,6 +305,12 @@
         if ($decl instanceof CfgType\Mixed_) {
             return self::mixed();
         }
+        if ($decl instanceof CfgType\Nullable) {
+            return (new self(self::TYPE_UNION, [
+                self::fromTypeDecl($decl->subtype),
+                new self(self::TYPE_NULL),
+            ]))->simplify();
+        }
 
         if ($decl instanceof CfgType\Union_) {
             $subs = [];
