--- vendor/ircmaxell/php-types/lib/PHPTypes/Type.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/Type.php
@@ -250,13 +250,13 @@ class Type
         }
         switch ($kind) {
             case 'var':
-                if (preg_match('(@var\s+(\S+))', $comment, $match)) {
+                if (preg_match('(@var\s+([^\s*][^\s]*))', $comment, $match)) {
                     return self::fromDecl($match[1]);
                 }
 
                 break;
             case 'return':
-                if (preg_match('(@return\s+(\S+))', $comment, $match)) {
+                if (preg_match('(@return\s+([^\s*][^\s]*))', $comment, $match)) {
                     return self::fromDecl($match[1]);
                 }
 
