--- vendor/ircmaxell/php-types/lib/PHPTypes/InternalArgInfo.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/InternalArgInfo.php
@@ -6705,7 +6705,7 @@
         'SNMP::walk' => ['', 'object_id' => '', 'suffix_as_key' => 'bool', 'max_repetitions' => 'int', 'non_repeaters' => 'int'],
         'snmpwalkoid' => ['array', 'hostname' => 'string', 'community' => 'string', 'object_id' => 'string', 'timeout=' => 'int', 'retries=' => 'int'],
         'SoapClient::__call' => ['', 'function_name' => 'string', 'arguments' => 'string'],
-        'SoapClient::__doRequest' => ['string', 'request' => 'string', 'action' => 'string', 'version' => 'int', 'one_way=' => 'int'],
+        'SoapClient::__doRequest' => ['?string', 'request' => 'string', 'location' => 'string', 'action' => 'string', 'version' => 'int', 'oneWay=' => 'bool'],
         'SoapClient::__getCookies' => ['array'],
         'SoapClient::__getFunctions' => ['array'],
         'SoapClient::__getLastRequestHeaders' => ['string'],
