--- vendor/ircmaxell/php-types/lib/PHPTypes/TypeReconstructor.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/TypeReconstructor.php
@@ -253,6 +253,12 @@ class TypeReconstructor
             case 'Expr_Yield':
             case 'Expr_Include':
                 // TODO: we may be able to determine these...
                 return false;
+            case 'Expr_FirstClassCallable':
+                if (\PHPCfg\Op\Expr\FirstClassCallable::KIND_METHOD === $op->kind) {
+                    return [Type::array()];
+                }
+
+                return [Type::string()];
         }
 
         throw new \LogicException('Unknown variable op found: '.$op->getType());
