--- vendor/ircmaxell/php-types/lib/PHPTypes/Type.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/Type.php
@@ -366,6 +366,9 @@
 
             return new self(self::TYPE_ARRAY, [$type]);
         }
+        if (preg_match('/^(list|array)\s*</i', trim($decl))) {
+            return new self(self::TYPE_ARRAY);
+        }
         $regex = '(^([a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*\\\\)*[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*$)';
         if (! preg_match($regex, $decl)) {
             throw new \RuntimeException("Unknown type declaration found: {$decl}");
