--- vendor/ircmaxell/php-cfg/lib/PHPCfg/Func.php
+++ vendor/ircmaxell/php-cfg/lib/PHPCfg/Func.php
@@ -33,6 +33,8 @@
 
     const FLAG_CLOSURE = 0x80;
 
+    const FLAG_READONLY = 0x100;
+
     /** @var string */
     public $name;
 
--- vendor/ircmaxell/php-cfg/lib/PHPCfg/Parser.php
+++ vendor/ircmaxell/php-cfg/lib/PHPCfg/Parser.php
@@ -513,7 +513,8 @@
     {
         $this->script->functions[] = $func = new Func(
             $node->namespacedName->toString(),
-            $node->byRef ? Func::FLAG_RETURNS_REF : 0,
+            ($node->byRef ? Func::FLAG_RETURNS_REF : 0)
+                | ($node->getAttribute('compilerReadonlyFunction', false) ? Func::FLAG_READONLY : 0),
             $this->parseTypeNode($node->returnType),
             null
         );
@@ -1218,6 +1219,9 @@
         $flags = Func::FLAG_CLOSURE;
         $flags |= $expr->byRef ? Func::FLAG_RETURNS_REF : 0;
         $flags |= $expr->static ? Func::FLAG_STATIC : 0;
+        if ($expr->getAttribute('compilerReadonlyFunction', false)) {
+            $flags |= Func::FLAG_READONLY;
+        }
 
         $this->script->functions[] = $func = new Func(
             '{anonymous}#'.++$this->anonId,
@@ -1238,6 +1242,9 @@
         $flags = Func::FLAG_CLOSURE;
         $flags |= $expr->byRef ? Func::FLAG_RETURNS_REF : 0;
         $flags |= $expr->static ? Func::FLAG_STATIC : 0;
+        if ($expr->getAttribute('compilerReadonlyFunction', false)) {
+            $flags |= Func::FLAG_READONLY;
+        }
 
         $this->script->functions[] = $func = new Func(
             '{anonymous}#'.++$this->anonId,
