--- vendor/ircmaxell/php-types/lib/PHPTypes/InternalArgInfo.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/InternalArgInfo.php
@@ -1412,7 +1412,7 @@
         'ErrorException::__toString' => ['string'],
         'Error::getCode' => ['int'],
         'Error::getFile' => ['string'],
-        'error_get_last' => ['array'],
+        'error_get_last' => ['array|null'],
         'Error::getLine' => ['int'],
         'Error::getMessage' => ['string'],
         'Error::getPrevious' => ['Throwable|Error'],
