--- vendor/ircmaxell/php-types/lib/PHPTypes/Type.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/Type.php
@@ -359,7 +359,7 @@
         }
         $regex = '(^([a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*\\\\)*[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*$)';
         if (! preg_match($regex, $decl)) {
-            throw new \RuntimeException("Unknown type declaration found: ${decl}");
+            throw new \RuntimeException("Unknown type declaration found: {$decl}");
         }
 
         return new self(self::TYPE_OBJECT, [], $decl);
@@ -488,7 +488,7 @@
                 throw new \RuntimeException('Unmatched braces?');
             }
             if (! in_array(substr($decl, 0, 1), ['|', '&'], true)) {
-                throw new \RuntimeException("Unknown position of combinator: ${decl}");
+                throw new \RuntimeException("Unknown position of combinator: {$decl}");
             }
             $right = self::fromDecl(substr($decl, 1));
             $combinator = substr($decl, 0, 1);
@@ -503,10 +503,10 @@
             } elseif ($orPos !== false && $andPos !== false) {
                 $pos = min($orPos, $andPos);
             } else {
-                throw new \RuntimeException("No combinator found: ${decl}");
+                throw new \RuntimeException("No combinator found: {$decl}");
             }
             if ($pos === 0) {
-                throw new \RuntimeException("Unknown position of combinator: ${decl}");
+                throw new \RuntimeException("Unknown position of combinator: {$decl}");
             }
             $left = self::fromDecl(substr($decl, 0, $pos));
             $right = self::fromDecl(substr($decl, $pos + 1));
@@ -519,7 +519,7 @@
             return new self(self::TYPE_INTERSECTION, [$left, $right]);
         }
 
-        throw new \RuntimeException("Unknown combinator ${combinator}");
+        throw new \RuntimeException("Unknown combinator {$combinator}");
     }
 }
--- vendor/ircmaxell/php-types/lib/PHPTypes/State.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/State.php
@@ -221,7 +221,7 @@
                     $map[$mapped][$name] = $class;
                 }
             } else {
-                echo "Could not find parent ${extends}\n";
+                echo "Could not find parent {$extends}\n";
             }
         }
         $this->classResolves = $map;
