--- vendor/ircmaxell/php-types/lib/PHPTypes/TypeReconstructor.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/TypeReconstructor.php
@@ -731,7 +731,7 @@
         if ($type instanceof Op\Type\Nullable) {
-            (new Type(Type::TYPE_UNION, [
+            return (new Type(Type::TYPE_UNION, [
                 $this->resolveOpType($type->subtype),
                 new Type(Type::TYPE_NULL),
             ]))->simplify();
         } elseif ($type instanceof Op\Type\Void) {
             return Type::null();
