--- vendor/ircmaxell/php-cfg/lib/PHPCfg/Parser.php
+++ vendor/ircmaxell/php-cfg/lib/PHPCfg/Parser.php
@@ -783,6 +783,7 @@
                 'Expr_AssignOp_Mul' => Op\Expr\BinaryOp\Mul::class,
                 'Expr_AssignOp_Plus' => Op\Expr\BinaryOp\Plus::class,
                 'Expr_AssignOp_Pow' => Op\Expr\BinaryOp\Pow::class,
+                'Expr_AssignOp_Coalesce' => Op\Expr\BinaryOp\Coalesce::class,
                 'Expr_AssignOp_ShiftLeft' => Op\Expr\BinaryOp\ShiftLeft::class,
                 'Expr_AssignOp_ShiftRight' => Op\Expr\BinaryOp\ShiftRight::class,
             ][$expr->getType()];
