--- vendor/ircmaxell/php-cfg/lib/PHPCfg/Parser.php
+++ vendor/ircmaxell/php-cfg/lib/PHPCfg/Parser.php
@@ -1490,8 +1490,26 @@
     /**
      * @param Expr\List_|Expr\Array_ $expr
      */
+    protected function isEmptyListExpr($expr): bool
+    {
+        foreach ($expr->items as $item) {
+            if (null !== $item) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    /**
+     * @param Expr\List_|Expr\Array_ $expr
+     */
     protected function parseListAssignment($expr, Operand $rhs)
     {
+        if ($this->isEmptyListExpr($expr)) {
+            throw new \CompileError('Cannot use empty list');
+        }
+
         $attributes = $this->mapAttributes($expr);
         foreach ($expr->items as $i => $item) {
             if (null === $item) {
