--- vendor/ircmaxell/php-cfg/lib/PHPCfg/Parser.php
+++ vendor/ircmaxell/php-cfg/lib/PHPCfg/Parser.php
@@ -39,6 +39,9 @@
 
     protected $astTraverser;
 
+    /** @var AstVisitor\MagicStringResolver */
+    protected $magicStringResolver;
+
     protected $fileName;
 
     /** @var FuncContext */
@@ -63,7 +66,8 @@
         $this->astTraverser = $astTraverser;
         $this->astTraverser->addVisitor(new AstVisitor\NameResolver());
         $this->astTraverser->addVisitor(new AstVisitor\LoopResolver());
-        $this->astTraverser->addVisitor(new AstVisitor\MagicStringResolver());
+        $this->magicStringResolver = new AstVisitor\MagicStringResolver();
+        $this->astTraverser->addVisitor($this->magicStringResolver);
     }
 
     /**
@@ -83,6 +87,7 @@
     public function parseAst($ast, $fileName): Script
     {
         $this->fileName = $fileName;
+        $this->magicStringResolver->beginCompilationUnit($fileName);
         $ast = $this->astTraverser->traverse($ast);
 
         $this->script = $script = new Script();
