diff --git vendor/ircmaxell/php-cfg/lib/PHPCfg/Parser.php vendor/ircmaxell/php-cfg/lib/PHPCfg/Parser.php
--- vendor/ircmaxell/php-cfg/lib/PHPCfg/Parser.php
+++ vendor/ircmaxell/php-cfg/lib/PHPCfg/Parser.php
@@ -1353,8 +1353,15 @@
 
     protected function parseExpr_New(Expr\New_ $expr)
     {
+        if ($expr->class instanceof Stmt\Class_) {
+            $this->parseStmt_Class($expr->class);
+            $class = $this->readVariable($this->parseExprNode($expr->class->namespacedName));
+        } else {
+            $class = $this->readVariable($this->parseExprNode($expr->class));
+        }
+
         return new Op\Expr\New_(
-            $this->readVariable($this->parseExprNode($expr->class)),
+            $class,
             $this->parseExprList($expr->args, self::MODE_READ),
             $this->mapAttributes($expr)
         );
