--- vendor/ircmaxell/php-llvm/lib/LLVMAbstract/Context.php
+++ vendor/ircmaxell/php-llvm/lib/LLVMAbstract/Context.php
@@ -94,14 +94,17 @@ class Context implements CoreContext {
     }
 
     public function functionType(CoreType $returnType, bool $isVarArgs, CoreType ... $parameters): CoreFunctionType {
-        $paramWrapper = $this->llvm->lib->makeArray(
-            LLVMTypeRef_ptr::class,
-            array_map(
-                function(Type $type) {
-                    return $type->type;
-                }, 
-                $parameters
-            )
-        );
+        $paramWrapper = null;
+        if (count($parameters) > 0) {
+            $paramWrapper = $this->llvm->lib->makeArray(
+                LLVMTypeRef_ptr::class,
+                array_map(
+                    function(Type $type) {
+                        return $type->type;
+                    },
+                    $parameters
+                )
+            );
+        }
         return $this->llvm->factory->type(
             $this, 
             $this->llvm->lib->LLVMFunctionType(
@@ -116,14 +119,17 @@ class Context implements CoreContext {
     }
 
     public function structType(bool $packed, CoreType ... $elements): CoreType {
-        $elementWrapper = $this->llvm->lib->makeArray(
-            LLVMTypeRef_ptr::class,
-            array_map(
-                function(Type $type) {
-                    return $type->type;
-                }, 
-                $elements
-            )
-        );
+        $elementWrapper = null;
+        if (count($elements) > 0) {
+            $elementWrapper = $this->llvm->lib->makeArray(
+                LLVMTypeRef_ptr::class,
+                array_map(
+                    function(Type $type) {
+                        return $type->type;
+                    },
+                    $elements
+                )
+            );
+        }
         return $this->llvm->factory->type(
             $this,
