--- vendor/ircmaxell/php-types/lib/PHPTypes/Type.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/Type.php
@@ -304,6 +304,10 @@
                 new self(self::TYPE_NULL),
             ]))->simplify();
         }
+        if ($decl instanceof CfgType\Reference) {
+            return self::fromDecl($decl->declaration->value);
+        }
+
         throw new \LogicException('Unsupported declaration type: '.get_class($decl));
     }
 
