--- vendor/ircmaxell/php-types/lib/PHPTypes/TypeReconstructor.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/TypeReconstructor.php
@@ -205,6 +205,7 @@
             case 'Expr_BinaryOp_Mod':
             case 'Expr_BinaryOp_ShiftLeft':
             case 'Expr_BinaryOp_ShiftRight':
+            case 'Expr_BinaryOp_Spaceship':
             case 'Expr_Cast_Int':
             case 'Expr_Print':
                 return [Type::int()];
