--- vendor/ircmaxell/php-types/lib/PHPTypes/Type.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/Type.php
@@ -342,6 +342,9 @@
                 new self(self::TYPE_NULL),
             ]))->simplify();
         }
+        if (preg_match('/^array\\{/i', $decl)) {
+            return new self(self::TYPE_ARRAY);
+        }
         switch (strtolower($decl)) {
             case 'boolean':
             case 'bool':
