--- vendor/ircmaxell/php-llvm/lib/LLVMAbstract/Context.php
+++ vendor/ircmaxell/php-llvm/lib/LLVMAbstract/Context.php
@@ -94,41 +94,51 @@
     public function ppcfp128Type(): CoreType {
         return $this->llvm->factory->type($this, $this->llvm->lib->LLVMPPCFP128TypeInContext($this->context));
     }
 
     public function functionType(CoreType $returnType, bool $isVarArgs, CoreType ... $parameters): CoreFunctionType {
-        $paramWrapper = $this->llvm->lib->makeArray(
-            LLVMTypeRef_ptr::class,
-            array_map(
-                function(Type $type) {
-                    return $type->type;
-                }, 
-                $parameters
-            )
-        );
+        $paramWrapper = null;
+        if (count($parameters) > 0) {
+            $paramTypes = [];
+            foreach ($parameters as $type) {
+                $paramTypes[] = $type->type;
+            }
+            $paramWrapper = $this->llvm->lib->makeArray(
+                LLVMTypeRef_ptr::class,
+                $paramTypes
+            );
+        }
         return $this->llvm->factory->type(
             $this, 
             $this->llvm->lib->LLVMFunctionType(
                 $returnType->type,
                 $paramWrapper,
                 count($parameters),
                 // LLVM is stupid, and even though the type is declared LLVMBool, it's not, and is a normal "1/0" bool instead of the weird reversed...
                 $isVarArgs ? 1 : 0
             )
         );
     }
 
     public function structType(bool $packed, CoreType ... $elements): CoreType {
-        $elementWrapper = $this->llvm->lib->makeArray(
-            LLVMTypeRef_ptr::class,
-            array_map(
-                function(Type $type) {
-                    return $type->type;
-                }, 
-                $elements
-            )
-        );
+        $elementWrapper = null;
+        if (count($elements) > 0) {
+            $elementTypes = [];
+            foreach ($elements as $type) {
+                $elementTypes[] = $type->type;
+            }
+            $elementWrapper = $this->llvm->lib->makeArray(
+                LLVMTypeRef_ptr::class,
+                $elementTypes
+            );
+        }
         return $this->llvm->factory->type(
             $this,
             $this->llvm->lib->LLVMStructTypeInContext(
                 $this->context,
                 $elementWrapper,
                 count($elements),
                 $this->llvm->toBool($packed)
             )
         );
     }
--- vendor/ircmaxell/php-llvm/lib/LLVMAbstract/Type/Struct.php
+++ vendor/ircmaxell/php-llvm/lib/LLVMAbstract/Type/Struct.php
@@ -13,16 +13,21 @@
         $this->type = $type;
         $this->name = $name;
     }
 
     public function setBody(bool $packed, CoreType ... $elements): void {
+        $elementTypes = [];
+        foreach ($elements as $type) {
+            $elementTypes[] = $type->type;
+        }
         $elementWrapper = $this->llvm->lib->makeArray(
             LLVMTypeRef_ptr::class,
-            array_map(
-                function(Type $type) {
-                    return $type->type;
-                }, 
-                $elements
-            )
+            $elementTypes
         );
         $this->llvm->lib->LLVMStructSetBody(
             $this->type,
             $elementWrapper,
             count($elements),
             $this->llvm->toBool($packed)
         );
     }
