--- vendor/ircmaxell/php-types/lib/PHPTypes/Type.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/Type.php
@@ -153,6 +153,11 @@
         return self::makeCachedType(self::TYPE_NULL);
     }
 
+    public static function never(): self
+    {
+        return self::makeCachedType(self::TYPE_NULL);
+    }
+
     public static function object(): self
     {
         return self::makeCachedType(self::TYPE_OBJECT);
@@ -295,6 +300,9 @@
         if ($decl instanceof CfgType\Literal) {
             return self::fromDecl($decl->name);
         }
+        if ($decl instanceof CfgType\Never_) {
+            return self::never();
+        }
         if ($decl instanceof CfgType\Mixed_) {
             return self::mixed();
         }
@@ -363,6 +371,8 @@
             case 'null':
             case 'void':
                 return new self(self::TYPE_NULL);
+            case 'never':
+                return self::never();
             case 'numeric':
                 return self::fromDecl('int|float');
         }

--- vendor/ircmaxell/php-types/lib/PHPTypes/TypeReconstructor.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/TypeReconstructor.php
@@ -775,6 +775,8 @@
             return Type::null();
         } elseif ($type instanceof Op\Type\Reference) {
             // todo
             return Type::fromDecl($type->declaration->value);
         } elseif ($type instanceof Op\Type\Literal) {
             return Type::fromDecl($type->name);
+        } elseif ($type instanceof Op\Type\Never_) {
+            return Type::never();
         }
 
         throw new \LogicException('Unknown Op\\Type provided: '.get_class($type));
