--- vendor/ircmaxell/php-cfg/lib/PHPCfg/AstVisitor/MagicStringResolver.php
+++ vendor/ircmaxell/php-cfg/lib/PHPCfg/AstVisitor/MagicStringResolver.php
@@ -76,8 +76,6 @@ class MagicStringResolver extends NodeVisitorAbstract
             if (! empty($this->methodStack)) {
                 return new Node\Scalar\String_(end($this->methodStack), $node->getAttributes());
             }
-        } elseif ($node instanceof Node\Scalar\MagicConst\Line) {
-            return new Node\Scalar\LNumber($node->getLine(), $node->getAttributes());
         }
     }
 
