--- vendor/ircmaxell/php-types/lib/PHPTypes/TypeReconstructor.php
+++ vendor/ircmaxell/php-types/lib/PHPTypes/TypeReconstructor.php
@@ -384,6 +384,11 @@ class TypeReconstructor
         return false;
     }
 
+    protected function resolveOp_Expr_Cast_Unset(Operand $var, Op\Expr\Cast\Unset_ $op, SplObjectStorage $resolved)
+    {
+        return [Type::null()];
+    }
+
     protected function resolveOp_Expr_Clone(Operand $var, Op\Expr\Clone_ $op, SplObjectStorage $resolved)
     {
         if ($resolved->contains($op->expr)) {
