--- vendor/ircmaxell/php-cfg/lib/PHPCfg/Parser.php
+++ vendor/ircmaxell/php-cfg/lib/PHPCfg/Parser.php
@@ -651,7 +651,15 @@
 
     protected function parseStmt_TraitUse(Stmt\TraitUse $node)
     {
-        // TODO
+        $traits = [];
+        foreach ($node->traits as $traitName) {
+            $traits[] = $this->parseExprNode($traitName);
+        }
+        $this->block->children[] = new Op\Stmt\TraitUse(
+            $traits,
+            $node->adaptations,
+            $this->mapAttributes($node)
+        );
     }
 
     protected function parseStmt_TryCatch(Stmt\TryCatch $node)
--- vendor/ircmaxell/php-cfg/lib/PHPCfg/Op/Stmt/TraitUse.php
+++ vendor/ircmaxell/php-cfg/lib/PHPCfg/Op/Stmt/TraitUse.php
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PHPCfg\Op\Stmt;
+
+use PHPCfg\Op\Stmt;
+use PHPCfg\Operand;
+
+class TraitUse extends Stmt
+{
+    /** @var Operand[] */
+    public $traits;
+
+    /** @var \PhpParser\Node\Stmt\TraitUseAdaptation[] */
+    public $adaptations;
+
+    public function __construct(array $traits, array $adaptations = [], array $attributes = [])
+    {
+        parent::__construct($attributes);
+        $this->traits = $traits;
+        $this->adaptations = $adaptations;
+    }
+
+    public function getVariableNames(): array
+    {
+        return [];
+    }
+}
