diff --git vendor/ircmaxell/php-cfg/lib/PHPCfg/Op/Expr/Assertion.php vendor/ircmaxell/php-cfg/lib/PHPCfg/Op/Expr/Assertion.php
index 1111111..2222222 100644
--- vendor/ircmaxell/php-cfg/lib/PHPCfg/Op/Expr/Assertion.php
+++ vendor/ircmaxell/php-cfg/lib/PHPCfg/Op/Expr/Assertion.php
@@ -17,7 +17,7 @@ use PHPCfg\Operand;
 
 class Assertion extends Expr
 {
-    public $read;
+    public $expr;
 
     public $assertion;
