--- vendor/ircmaxell/php-llvm/lib/LLVMAbstract/Context.php	2026-05-27 18:51:49.221534016 +0000
+++ vendor/ircmaxell/php-llvm/lib/LLVMAbstract/Context.php	2026-05-27 18:51:49.221590214 +0000
@@ -118,14 +118,17 @@
     }
 
     public function structType(bool $packed, CoreType ... $elements): CoreType {
-        $elementTypes = [];
-        foreach ($elements as $type) {
-            $elementTypes[] = $type->type;
+        $elementWrapper = null;
+        if (count($elements) > 0) {
+            $elementTypes = [];
+            foreach ($elements as $type) {
+                $elementTypes[] = $type->type;
+            }
+            $elementWrapper = $this->llvm->lib->makeArray(
+                LLVMTypeRef_ptr::class,
+                $elementTypes
+            );
         }
-        $elementWrapper = $this->llvm->lib->makeArray(
-            LLVMTypeRef_ptr::class,
-            $elementTypes
-        );
         return $this->llvm->factory->type(
             $this,
             $this->llvm->lib->LLVMStructTypeInContext(
