diff --git vendor/ircmaxell/php-cfg/lib/PHPCfg/AstVisitor/LoopResolver.php vendor/ircmaxell/php-cfg/lib/PHPCfg/AstVisitor/LoopResolver.php
--- vendor/ircmaxell/php-cfg/lib/PHPCfg/AstVisitor/LoopResolver.php
+++ vendor/ircmaxell/php-cfg/lib/PHPCfg/AstVisitor/LoopResolver.php
@@ -68,6 +68,13 @@
 
     protected function resolveStack(Node $node, array $stack)
     {
+        if ([] === $stack) {
+            $keyword = 'Stmt_Break' === $node->getType() ? 'break' : 'continue';
+
+            throw new \LogicException(
+                sprintf("'%s' not in the 'loop' or 'switch' context", $keyword)
+            );
+        }
         if (! $node->num) {
             return new Goto_(end($stack), $node->getAttributes());
         }
